src/identify/libp2p_identify.erl

```diff
@@ -15,10 +15,7 @@
 -spec from_map(identify_map(), libp2p_crypto:sig_fun()) -> identify().
 from_map(Map, SigFun) ->
     G = fun(K) ->
-                case maps:get(K, Map, no_entry) of
-                    no_entry -> error(badarg);
-                    Value -> Value
-                end
+                maps:get(K, Map)
```

Comment:
Can you just get rid of the fun and use maps:get directly?

Reply:
my fingers are bleeding from all that typing :-)

Reply:
done